### DISCUSSION TITLE
TextureModeRtmpActivity

### DISCUSSION THREAD

**Discussion by yuanyijj** [pedroSG94/RootEncoder#1316] [Q&A]
使用  <com.pedro.library.view.AutoFitTextureView  推流的时候 怎么只推了声音呢    推摄像头的流还需要怎么设置呢

**Comment by pedroSG94** [comment]
Hello,

Frist of all, test with the example in the app sample without modifications to know if it is working fine. You can use Texture mode example.
After that, if you still have the same problem share me a logcat of Texture mode example with the error.
